CONTRIBUTING.md

```diff
@@ -104,6 +104,30 @@ webapp:dev: If you didn't try to log in, you can safely ignore this email.
   ](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
 - Be sure to fill the PR Template accordingly.
 
+## Adding changesets
+
+We use changesets to track changes and structure the descriptions of various changes in this project in a format that is readable/comprehensible by everyone.
+
+Since Trigger.dev is a monorepo that contains various packages, it is important that we integrate this workflow.
+
+Before you create a Pull Request be sure to use the command below to add a changeset describing what you have done.
+
+```shell
+pnpm changeset:add
```

Comment:
I think this needs to be `pnpm run changeset:add`, right? Also make sure to note that this should be run from the root of the repo